engines/bladerunner/vqa_decoder.cpp

```diff
@@ -1113,18 +1113,36 @@ void VQADecoder::VQAVideoTrack::VPTRWriteBlock(Graphics::Surface *surface, unsig
 		const uint8 *src_p = block_src;
 
 		for (uint y = _blockH; y != 0; --y) {
-			for (uint x = _blockW; x != 0; --x) {
-				vqaColor = READ_LE_UINT16(src_p);
+#ifdef SCUMMVM_NEON
+			if (_blockW == 4) {
```

Comment:
This needs to be checked at runtime as well using `g_system->hasFeature(OSystem::kFeatureCpuNEON)`.

Reply:
Done